code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
+	var/permckey = "" //The perm ckey, never removed essentially.
+	var/permuser = "" //Same as above!
+	var/uID = "" //The UNIQUE id, made by putting part of the ckey and the ID together. Used internally in code.
+	var/active = "No" //Is the adminhelp active, eg admin responded? This is the same as above, it makes life easier.
+	var/logs = list() //The logs of the adminhelp.
+	var/replying = FALSE //Is someone responding to the adminhelp?
+	var/mob //The mob adminhelping mob.
+
+/client/proc/list_ahelps(user, resolved)
+	if(!check_rights(R_ADMIN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	if(resolved)
+		user << "Resolved Ahelps:"
+		for(var/datum/adminticket/T in admintickets)
```

Comment:
Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)